src/Tests.Dapper.Oracle/Tests.Dapper.Oracle.csproj

```diff
@@ -9,6 +9,7 @@
   <ItemGroup>
     <PackageReference Include="FluentAssertions" Version="5.3.0" />
     <PackageReference Include="Oracle.ManagedDataAccess.Core" Version="2.18.3" />
+    <PackageReference Include="System.Data.SqlClient" Version="4.8.1" />
```

Comment:
Just curious - why add SqlClient here?

Reply:
That package was added because an existing test contains a reference to SqlCommand. When I removed .NET 4.5.2, the test project no longer recognized System.Data.SqlClient.

`       
[Fact]
public void WrongTypeConnectionWillThrowException()
{
	var cmd = new SqlCommand();
	testObject.Add("Foo", "Bar");
	Action act = () => testObject.AddParam(cmd);
	act.Should().Throw<NotSupportedException>();
}         
`

Reply:
Ok.  It's fine as long as it is only a test project.